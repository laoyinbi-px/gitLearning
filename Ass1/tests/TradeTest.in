add trader1 100
add trader2 100
add trader3 100
import trader1 ABC 100
buy trader2 ABC 10 7.5
sell trader1 ABC 100 6.5
TRADES TRADER trader1
TRADES TRADER trader3
TRADES TRADER trader2
exit