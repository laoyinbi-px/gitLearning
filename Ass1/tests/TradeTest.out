$ Success.
$ Success.
$ Success.
$ Trader now has 100.00 units of ABC.
$ No trades could be made, order added to buy book.
$ Product sold in part, trades as follows:
trader1->trader2: 10.00xABC for $7.50.
$ trader1->trader2: 10.00xABC for $7.50.
$ No trades have been completed by trader.
$ trader1->trader2: 10.00xABC for $7.50.
$ Have a nice day.